File: openbis/source/java/ch/systemsx/cisd/openbis/public/resources/api/v3/as/dto/event/id/IEventId.js
/**
 * Holds information that uniquely identifies an event in openBIS.
 *
 * @author pkupczyk
 */
define([ "stjs", "as/dto/common/id/IObjectId" ], function(stjs, IObjectId) {
	var IEventId = function() {
	};
	stjs.extend(IEventId, null, [ IObjectId ], null, {});
	return IEventId;
})

File: openbis_api/source/java/ch/ethz/sis/openbis/generic/asapi/v3/dto/event/id/IEventId.java
package ch.ethz.sis.openbis.generic.asapi.v3.dto.event.id;

import ch.ethz.sis.openbis.generic.asapi.v3.dto.common.id.IObjectId;
import ch.systemsx.cisd.base.annotation.JsonObject;

/**
 * Holds information that uniquely identifies an event in openBIS.
 *
 * @author pkupczyk
 */
@JsonObject("as.dto.event.id.IEventId")
public interface IEventId extends IObjectId
{

}
